README.MD

```diff
@@ -1,34 +1,51 @@
 # An Algorithm to Estimate the Two-Way Fixed Effect Model
 
-This repo contains the ado file needed to run the Two-way fixed effect model in Stata. Run the ado file first, or place it in the ado-files folder in Stata.
+This repo contains the ado file needed to run the Two-way fixed effect model in Stata. Run the ado file first or place it in the ado-files folder in Stata.
 
-twowayset defines the two variables with the household id and the time id. For example:
+twowaywrapper is a command that runs the whole algorithm with many options for the user, the syntaxis is the following:
 ```
-.twowayset hhid tid
-.twowayset hhid tid w
+.varlist(numeric ts fv) [if] [in], [,ABSorb(varlist min=2 max=3) DROP GENerate(name)] [, NEWVars(name) REPLACE NOPROJ] [, VCE(name)]
 ```
```

Comment:
It doesn't make sense to add any two of the following three options:
[, NEWVars(name) REPLACE NOPROJ]
Am I right?

In that case, it'd be best if the code crashed and returned an explanation saying that it doesn't make sense when the user specifies two or more options.